Makefile

```diff
@@ -262,7 +262,7 @@ push-openstack-populator-image: build-openstack-populator-image
 
 build-vsphere-xcopy-volume-populator-image: check_container_runtime
 	$(eval VSPHERE_XCOPY_VOLUME_POPULATOR_IMAGE=$(REGISTRY)/$(REGISTRY_ORG)/vsphere-xcopy-volume-populator:$(REGISTRY_TAG))
-	$(CONTAINER_CMD) build -t $(VSPHERE_XCOPY_VOLUME_POPULATOR_IMAGE) -f build/vsphere-xcopy-volume-populator/Containerfile cmd/vsphere-xcopy-volume-populator
+	$(CONTAINER_CMD) build -t $(VSPHERE_XCOPY_VOLUME_POPULATOR_IMAGE) -f build/vsphere-xcopy-volume-populator/Containerfile .
```

Comment:
Why do we need to the whole repo in the build context?

Reply:
When I restrained the context on the github action only to the `cmd/vsphere-xcopy-volume-populator` it could not access the container file `build/vsphere-xcopy-volume-populator/Containerfile`